src/components/AdminSettings/MatterbridgeIntegration.vue

```diff
@@ -146,7 +146,7 @@ export default {
 		async enableMatterbridgeApp() {
 			if (OC.PasswordConfirmation.requiresPasswordConfirmation()) {
 				OC.PasswordConfirmation.requirePasswordConfirmation(this.enableMatterbridgeAppCallback, {}, () => {
-					showError(t('spreed', 'An error occurred while installing the Matterbridge app'))
+					showError(t('spreed', 'An error occurred while installing the Matterbridge app.'))
```

Comment:
I feel trolled now.

cc @rakekniven @Valdnet https://github.com/nextcloud/spreed/pull/9288

I'm still waiting for the answer:
> My question remains, when should the dot at the end of a full sentence be skipped and why?

Reply:
Sorry for that mate 🙏